kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OnlineShopUser.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.good.patterns.challenges;
+
+public class OnlineShopUser implements Customer
```

Comment:
Jeśli to ma być prawdziwa klasa, której obiekty mają niezmienny stan (czyli immutable) to klasa też powinna być jako final

Reply:
Poprawione